### PR TITLE
Update PaintAction.js

### PR DESCRIPTION
**Pull request by nomnomnomyes**
some android tablets have paint editor bad touch point. i hope this might help the buggy android tablets without changing anything in the already working ipad and android tablets.

### CODE REVIEW

File: src/painteditor/PaintAction.js [hunk 1]
```diff
@@ -1131,8 +1131,6 @@ export default class PaintAction {
         pt2.x = pt.x;
         pt2.y = pt.y;
         var globalPoint = pt2.matrixTransform(Paint.root.getScreenCTM().inverse());
-        globalPoint.x = globalPoint.x / Paint.currentZoom;
-        globalPoint.y = globalPoint.y / Paint.currentZoom;
```
Comment (chrisgarrity) [inline src/painteditor/PaintAction.js]: Have you tried this along with changing the default zoom in the paint editor? (Pinch to zoom)

Comment (chrisgarrity) [reply src/painteditor/PaintAction.js]: Unfortunately removing these lines breaks the paint editor for all iPads. 

I'm closing this PR. I'll submit a separate PR that addresses it without breaking iOS

**Comment by chrisgarrity** [comment]
BTW, thanks for the pointers - with this we were able to fix the issue in: https://github.com/LLK/scratchjr/pull/170

**Comment by sparkguy888** [comment]
I have the same issue on Zenpad 10. May I ask how to replace this PaintAction.js file in my pad? I can't find the original file in my pad. Many thanks

**Comment by chrisgarrity** [comment]
@sparkguy888 we're currently testing a new version that includes this bug fix. Depending on what our testing turns up, and how long the app store approval process takes, there should be a fix out soon.

**Pull request closed without merging**